include/ignition/gui/GuiEvents.hh

```diff
@@ -323,6 +323,29 @@ namespace ignition
         /// \brief Private data pointer
         IGN_UTILS_IMPL_PTR(dataPtr)
       };
+
+      /// \brief Event which is called to broadcast the 2D coordinates of a
+      /// user's mouse hover within the scene.
+      class IGNITION_GUI_VISIBLE HoverOnScene : public QEvent
+      {
+        /// \brief Constructor
+        /// \param[in] _point The point at which the mouse is hovering within
+        /// the scene
+        public: explicit HoverOnScene(const math::Vector2i &_point);
+
+        /// \brief Unique type for this event.
+        static const QEvent::Type kType = QEvent::Type(QEvent::MaxUser - 13);
```

Comment:
Noob question, why `-13`? Can this be documented in the comments?

Reply:
Good idea to comment it, because it keeps coming up :sweat_smile: 

tl;dr: `gazebo::gui:GuiEvents` count up, `gui::GuiEvents` count down.

See context here: https://github.com/ignitionrobotics/ign-gui/pull/70#discussion_r436873992